src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\"</string>
+  <string name="error_validating_format">Could not validate \"%s\"</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all new versions</string>
+  <string name="ignore_this_update">Ignore this new version</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the one installed on your device.
+    You\'ll have to uninstall the application to install it.</string>
   <string name="incompatible_platforms_description_format">Your %1$s platform is not supported.
     Supported platforms: %2$s.</string>
   <string name="incompatible_sdk_description_format">Your %1$s (SDK version %2$d) is not supported. %3$s</string>
-  <string name="incompatible_sdk_max_description_format">Maximum SDK version is %d.</string>
-  <string name="incompatible_sdk_min_description_format">Minimum SDK version is %d.</string>
+  <string name="incompatible_sdk_max_description_format">No SDK version higher than %d is allowed.</string>
+  <string name="incompatible_sdk_min_description_format">No SDK version lower than %d is allowed.</string>
```

Comment:
How about "Install Android "version of SDK in question" or older/newer"?

Reply:
No, it's wrong to suggest to install specific Android version. Incompatibility information is enough.

It's impossible to get a human-readable Android version by SDK version, so I'm just displaying the information F-Droid provides.

"No SDK version higher than %d is allowed" — for me "SDK version higher than %d is not allowed" looks better.

Reply:
One is a garden-path, the other is an unmarked noun. How about "SDK %d is the latest allowed version"

Reply:
It's better to use "SDK version" instead of "SDK" (because "SDK version" is already used in `incompatible_sdk_description_format`).

"SDK version %d is the latest allowed version" is good as "Maximum SDK version is %d" replacement. But what about "Minimum SDK version is %d"?